config/samples_2016.yaml

```diff
@@ -1,5 +1,17 @@
 GLOBAL:
-  period: '2016'
+  era: '2016'
+  luminosity: 36.31
```

Comment:
this is full 2016 lumi - better not to put

Reply:
ok